services/apps/alcs/src/alcs/application-decision/application-boundary-amendment/application-boundary-amendment.controller.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Patch,
+  Post,
+  UseGuards,
+} from '@nestjs/common';
+import { ApiOAuth2 } from '@nestjs/swagger';
+import { Mapper } from 'automapper-core';
+import { InjectMapper } from 'automapper-nestjs';
+import * as config from 'config';
+import { ANY_AUTH_ROLE } from '../../../common/authorization/roles';
+import { RolesGuard } from '../../../common/authorization/roles-guard.service';
+import { UserRoles } from '../../../common/authorization/roles.decorator';
+import {
+  ApplicationBoundaryAmendmentDto,
+  CreateApplicationBoundaryAmendmentDto,
+  UpdateApplicationBoundaryAmendmentDto,
+} from './application-boundary-amendment.dto';
+import { ApplicationBoundaryAmendment } from './application-boundary-amendment.entity';
+import { ApplicationBoundaryAmendmentService } from './application-boundary-amendment.service';
+
+@ApiOAuth2(config.get<string[]>('KEYCLOAK.SCOPES'))
+@UseGuards(RolesGuard)
+@Controller('application-boundary-amendment')
+export class ApplicationBoundaryAmendmentController {
+  constructor(
+    private applicationBoundaryAmendmentService: ApplicationBoundaryAmendmentService,
+    @InjectMapper() private mapper: Mapper,
+  ) {}
+
+  @Get('/application/:fileNumber')
+  @UserRoles(...ANY_AUTH_ROLE)
```

Comment:
so, could anyone create amendments?  I thought that this feature was for Gabriel

Reply:
Confirmed with Ansley there are no role restrictions